libs/x-content/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -850,7 +850,7 @@ private XContentBuilder value(ToXContent value) throws IOException {
         return value(value, ToXContent.EMPTY_PARAMS);
     }
 
-    private XContentBuilder value(ToXContent value, ToXContent.Params params) throws IOException {
+    public XContentBuilder value(ToXContent value, ToXContent.Params params) throws IOException {
```

Comment:
I don't really see why making this public would be a problem given what it does and the other public methods nearby.  However, you should probably run this by somebody in the ES core infra team before merging.

There is a simple way to avoid making this public - see my next comment.